lib/src/test/java/uk/gov/di/ipv/cri/address/library/service/AuditServiceTest.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.di.ipv.cri.address.library.service;
+
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.model.SendMessageRequest;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.di.ipv.cri.address.library.domain.AuditEventTypes;
+import uk.gov.di.ipv.cri.address.library.exception.SqsException;
+import uk.gov.di.ipv.cri.address.library.models.AuditEvent;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class AuditServiceTest {
+    @Mock AmazonSQS mockSqs;
+    @Mock ConfigurationService mockConfigurationService;
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    private AuditService auditService;
+
+    @BeforeEach
+    void setup() {
+        when(mockConfigurationService.getSqsAuditEventQueueUrl())
+                .thenReturn("https://example-queue-url");
+
+        auditService = new AuditService(mockSqs, mockConfigurationService);
+    }
+
+    @Test
+    void shouldSendMessageToSqsQueue() throws JsonProcessingException, SqsException {
+        UUID sessionId = UUID.randomUUID();
+        String clientId = "client-id";
+        auditService.sendAuditEvent(AuditEventTypes.SESSION_CREATED, sessionId, clientId);
+
+        ArgumentCaptor<SendMessageRequest> sqsSendMessageRequestCaptor =
+                ArgumentCaptor.forClass(SendMessageRequest.class);
+        verify(mockSqs).sendMessage(sqsSendMessageRequestCaptor.capture());
+
+        assertEquals(
+                "https://example-queue-url", sqsSendMessageRequestCaptor.getValue().getQueueUrl());
```

Comment:
Please could you make this a const, so you don't have to repeating typing the literal string every where